File: src/classes/Telefone.java
package classes;

public class Telefone {
	private int id;
	private String numero;
	private Endereco endereco;
	private Parceiro parceiro;
	private String descricao;
	
	public String getNumero() {
		return numero;
	}
	public void setNumero(String numero) {
		this.numero = numero;
	}
	
	public Endereco getEndereco() {
		return endereco;
	}
	public void setEndereco(Endereco endereco) {
		this.endereco = endereco;
	}
	
	public Parceiro getParceiro() {
		return parceiro;
	}
	public void setParceiro(Parceiro parceiro) {
		this.parceiro = parceiro;
	}
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getDescricao() {
		return descricao;
	}
	public void setDescricao(String descricao) {
		this.descricao = descricao;
	}
}

File: API C#/ListaPublica/ListaPublica/Classes/Telefone.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ListaPublica.Classes
{
    public class Telefone
    {
        public int id { get; set; }
        public string numero { get; set; }
        public string descricao { get; set; }
        public Endereco endereco { get; set; }
        public Parceiro parceiro { get; set; }
    }
}
